pkg/router/functionHandler.go

```diff
@@ -289,8 +291,18 @@ func (roundTripper RetryingRoundTripper) RoundTrip(req *http.Request) (*http.Res
 
 		roundTripper.logger.Debug("request headers", zap.Any("headers", req.Header))
 
+		//Creating context for client
+		if roundTripper.timeout == 0 {
```

Comment:
oops, I missed this.
`roundTripper.timeout == 0` -> `roundTripper.timeout <= 0`

Reply:
Yes, I add only  `==` instead of `<=` because it was uint so can't be `< 0`. But now when I change `uint64` to `int` this change will also be required.